emcc.py

```diff
@@ -3558,7 +3568,8 @@ def preprocess_wasm2js_script():
   if options.emit_symbol_map:
     intermediate_debug_info -= 1
     if os.path.exists(wasm_target):
-      building.handle_final_wasm_symbols(wasm_file=wasm_target, symbols_file=symbols_file, debug_info=intermediate_debug_info)
+      with ToolchainProfiler.profile_block('handle_final_symbols'):
+        building.handle_final_wasm_symbols(wasm_file=wasm_target, symbols_file=symbols_file, debug_info=intermediate_debug_info)
       save_intermediate_with_wasm('symbolmap', wasm_target)
```

Comment:
In other places you've put the `save_intermediate` call inside the profile block, maybe do that here too for consistency?